src/data/datatools.ts

```diff
@@ -15,13 +15,16 @@ export function replaceAll(target: string, search: string, replacement: string):
 }
 
 function fieldValueToString(valueObject: IFieldValue): string {
+  if (valueObject.value == null) {
+    return "";
+  }
   if (valueObject.type === "ProcessHubDate") {
     return getFormattedDate(new Date(valueObject.value.toString()));
   } else if (valueObject.type === "ProcessHubDateTime") {
     const date: Date = new Date(valueObject.value.toString());
     return getFormattedDateTime(date) + " " + getFormattedTimeZoneOffset(date.getTimezoneOffset());
   } else {
-    return valueObject.value != null ? valueObject.value.toString() : "";
+    return valueObject.value.toString();
```

Comment:
Wouldn't it make sense to define a return string at the beginning, set its value in the different ifs and just return that string in the end? That would probably simplify possible debugging in the future.